server/sonar-web/src/main/js/apps/quality-gates/components/Details.js

```diff
@@ -85,14 +86,14 @@ export default class Details extends Component {
   }
 
   handleDeleteClick() {
-    const { qualityGate, onDelete } = this.props;
+    const { qualityGate, onDelete, organization } = this.props;
     const { router } = this.context;
-
+    const urlPrefix = organization ? '/organizations/' + organization.key : '';
     new DeleteView({
       qualityGate,
       onDelete: qualityGate => {
         onDelete(qualityGate);
-        router.replace('/quality_gates');
+        router.replace(urlPrefix + '/quality_gates');
```

Comment:
what about creating `getQualityGatesUrl` (with "s") the same way as `getQualityGateUrl`?